src/AspNetCore/Example/Microsoft.AspNetCore.rd.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Directives>
+	<Application>
+		<Assembly Name="System.Text.Json">
+			<Type Name="System.Text.Json.Serialization.Converters.DictionaryOfTKeyTValueConverter`3[[System.Collections.Generic.Dictionary`2[[System.String,System.Private.CoreLib],[System.Object,System.Private.CoreLib]],System.Private.CoreLib],[System.String,System.Private.CoreLib],[System.Object,System.Private.CoreLib]]" Dynamic="Required All" />
+		</Assembly>
+		<Assembly Name="Microsoft.AspNetCore.Mvc.Razor">
+			<Type Name="Microsoft.AspNetCore.Mvc.ApplicationParts.ConsolidatedAssemblyApplicationPartFactory" Dynamic="Required All" />
+		</Assembly>
+		<Assembly Name="Microsoft.AspNetCore.Mvc.Core" Dynamic="Required All">
+			<Type Name="Microsoft.Extensions.Internal.PropertyHelper">
+				<!--
+					Change WebApplication.Pages.ErrorModel to full type name of the ErrorModel
+					WebApplicationAssembly is name of your assembly where ErrorModel class is located.
+				-->
+				<!-- <Method Name="CallNullSafePropertyGetter" Dynamic="Required">
+					<GenericArgument Name="WebApplication.Pages.ErrorModel, WebApplicationAssembly" />
+					<GenericArgument Name="System.Boolean, System.Private.CoreLib" />
+				</Method> -->
```

Comment:
```suggestion
```

Generated code?

Reply:
Not quite. I plan to provide simple templates for starting with NativeAOT in https://github.com/kant2002/RdXmlLibrary. Create them from scratch is unnescessary pain for most. I can remove these lines obviously.

Reply:
I would remove.